net/lksctp-tools/Makefile

```diff
@@ -8,12 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lksctp-tools
-PKG_VERSION:=1.0.19
+PKG_VERSION:=1.0.21
 PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://codeload.github.com/sctp/lksctp-tools/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=9251b1368472fb55aaeafe4787131bdde4e96758f6170620bc75b638449cef01
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/sctp/lksctp-tools
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=0abcbd81d10be7986b462be879579e18f45fa44bcf6f61dd7624138dbb1fa779
```

Comment:
Why the switch? As per [contributing guidelines](https://github.com/openwrt/packages/blob/d52e588182f0906ee04e036a62ce88d6da57911d/CONTRIBUTING.md#package-sources-archives-and-repositories), source archive is preferred and downloading from a repo is the last resort.

Reply:
> source archive

I guess that means the one(s) uploaded by repo maintainers (for example [syncthing-source-v2.0.3.tar.gz](https://github.com/syncthing/syncthing/releases/download/v2.0.3/syncthing-source-v2.0.3.tar.gz)), not the codeload.

Actually people keep arguing about whether to use codeload or local tarball. Some people prefer codeload, while others prefer local tarball.